src/server/migrations/20211103150400_order_items.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('order_items', (table) => {
```

Comment:
The migration has to run after 20211105095000_products-table.js  so the name of the file has to change accordingly